posix.mak

```diff
@@ -864,23 +864,49 @@ test: $(ASSERT_WRITELN_BIN)_test test/next_version.sh all
 #     - The changelog generation script searches for all Ddoc files within the `changelog` folders
 #       and adds them to the generated changelog
 #
-# The changelog script is at https://github.com/dlang/tools/blob/master/changed.d
+#  The changelog script is at https://github.com/dlang/tools/blob/master/changed.d
+#
+#  The changelog generation has two targets:
+#
+#  a) Preview upcoming changes
+#
+#  			make -f posix.mak pending_changelog
+#
+#  	  This will look at changes up to `upstream/master` and can be used to
+#  	  preview the changelog locally.
+#
+#  b) Generate the changelog for an upcoming release
+#
+#  		make -f posix.mak prerelease_changelog
+#
+#   This will look at changes to upstream/stable and is run by the release manager.
```

Comment:
Seems like a lot of gratuitous whitespace in those comments.

Reply:
Yes, this change still used tabs. Good catch!